airflow/providers/slack/CHANGELOG.rst

```diff
@@ -24,12 +24,24 @@
 Changelog
 ---------
 
+7.2.0
+.....
+
+Features
+~~~~~~~~
+
+* ``Add general-purpose "notifier" concept to DAGs (#28569)``
```

Comment:
Interesting one. On one hand - it's worth having it, on the other - it won't be useful until 2.6.0 . Maybe worth mentioning it that it here?

Reply:
there is no harm in releasing it up front, it means that 2.6.0 users will be able to utalize it with older versions of slack provider but we can also decide not to release slack provider since this is the only feature in the release

Reply:
Yep. I guess no problem.

Reply:
we can wait and release this only if we have additional faeture/bug fix for Slack provider because my argument is valid only if between now and 2.6 there will be more releases to the provider

Reply:
I would say - let's release it now. No harm, and makes it easier to put it behind. This the same story as discussed before- the more un-released stuff we keep the more difficult is to deal with any problems that it might cause as side effects. I think there is a value in releasing all releasable stuff as soon as possible (even if it is not useful for now).